tools/composer.json

```diff
@@ -5,7 +5,8 @@
         "phpstan/phpstan-phpunit": "^2.0.6",
         "vimeo/psalm": "^6.11.0",
         "roave/backward-compatibility-check": "^8.13.0",
-        "roave/infection-static-analysis-plugin": "^1.37.0"
+        "roave/infection-static-analysis-plugin": "^1.38.0",
+        "phpstan/mutant-killer-infection-runner": "1.0.x-dev"
```

Comment:
Adding a note here: let's get a stable (even if `0.x`) release out, before going with this

Reply:
I'm not probably going to release this package, there's a bug that causes PHPStan not be called for some mutants and I don't know why. I talked to Maks at PHPers over the weekend and the official support in Infection should land very soon.

We will continue working on this, mostly for performance improvements, but there's no reason to delay the feature in Infection itself.

Reply:
UPD: https://github.com/Roave/BetterReflection/pull/1510